packages/tc-web/src/components/tc-segment-container/tc-segment-container.tsx

```diff
@@ -0,0 +1,33 @@
+// segment-container.tsx
+import { Component, Prop, h, Host } from "@stencil/core";
+
+@Component({
+  tag: "segment-container",
+  styleUrl: "tc-segment-container.css",
+  shadow: true,
+})
+export class SegmentContainer {
+  @Prop() segTitle: string;
```

Comment:
```suggestion
  @Prop() title: string;
```

the component is called SegmentContainer -> title would suffice to make it clear what it is

Reply:
hab es geändert, nun kann man es aufrufen mit title
@Prop({attribute:"title"}) segTitle: string;

Wieso?
[title is a reserved name](https://stackoverflow.com/questions/57675138/what-is-the-right-way-to-not-use-reserved-public-name-withing-web-component)

Reply:
hoi @Queentaker, ich glaube, title ist ein html attribut für accessibility. dürftes du wohl nicht als prop name verwenden. sorry, das wusste ich nicht, dann müsste man es wohl segmentTitle nennen (dann ist der name auch klar)